### ISSUE TITLE
ElementRef exposes setters for parentView, boundElementIndex. Accident?

### ISSUE THREAD

**Issue by matanlurey** [angular/angular#2440]
I assume parentView, boundElementIndex should not be modifiable, but the API exposes them.

@tbosch

**Comment by mhevery** [comment]
Blocked on https://github.com/Microsoft/TypeScript/issues/12 Probably won't fix any time soon. dropping priority.

**Comment by mhevery** [comment]
already fixed
